piqe_ocp_lib/tests/resources/test_ocp_machine_management.py

```diff
@@ -150,3 +150,26 @@ def test_get_machine_node_ref(self, get_test_objects):
         node_obj = node_obj.get_a_node(node_name)
         assert node_obj.kind == "Node"
         assert node_obj.metadata.name == node_name
+
+
+@pytest.mark.skip_if_not_provider("Azure")
+class TestOcpMachineHealthCheck:
+    def test_get_machine_health_checks(self, get_test_objects):
+        m_obj = get_test_objects.machine_health_check
+        machine_health_check_list = m_obj.get_machine_health_checks()
+        assert isinstance(machine_health_check_list, ResourceInstance)
+        assert machine_health_check_list.kind == "MachineHealthCheckList"
```

Comment:
do we have something like 'status' in the api response for assertion check

Reply:
This is only to check if the resource kind is `MachineHealthCheckList`.